.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "files.exclude": {
```

Comment:
Wrong file but reminder to only show filters for stores returned by product search

Alternatively, add endpoint to merchant API to get turned on merchants